wordpress-mobile/WordPress-iOS
# UL&S: TwoFAViewController: add VoiceOver support

Ref. #14009 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/369

This PR adds VoiceOver support to the TwoFAViewController. 

### To test:
Pre-requisite:  Have a Google account with WP 2fa enabled.

1. Enable the `.unifiedGoogle` feature flag
2. `rake dependencies`
3. Build and run on a device.
4. `Hey Siri turn VoiceOver on`
5. Log In > Continue with Google
6. When the page navigates to the Unified Auth 2fa view, it should read: "Authentication code. Textfield. Is editing. Authentication code. Character mode. Insertion point at start."
7. Two-finger swipe down to read the textfield again and the elements below it. The correct order is: "<reads textfield again>, Text me a code instead, button. Continue, dimmed, button. <reads numeric keypad>." (Optional: two finger swipe up, instead. It will read the instruction label, then read the other elements below the instructions).
8. Enter an invalid code. VoiceOver should focus on the error message and read it when the UI changes.

**Screenshot**
<a href="https://user-images.githubusercontent.com/1062444/89830063-3a7e9780-db21-11ea-9a94-8284c2674699.PNG"><img src="https://user-images.githubusercontent.com/1062444/89830063-3a7e9780-db21-11ea-9a94-8284c2674699.PNG" width="350" /></a>


PR submission checklist:
